todo_app/tests_e2e/test_e2e.py

```diff
@@ -46,6 +48,8 @@ def app_with_temp_board():
 
     # construct the new application
     application = app.create_app()
+    
+    application.config['LOGIN_DISABLED'] = (os.environ['LOGIN_DISABLED'] == 'True')
```

Comment:
Given you set this to environment variable to `'True'` just above, can't we just set this directly to True? I.e.
```suggestion
    application.config['LOGIN_DISABLED'] = True
```
Also, if we're acknowledging (as I think we are here!) that the environment variable above isn't set in time for the config to correctly pick it up, I think it would be clearer to just have this line setting the config to True, and skip setting the `LOGIN_DISABLED` environment variable at all